processmanager/manager.go

```diff
@@ -348,34 +347,6 @@ func (pm *ProcessManager) MaybeNotifyAPMAgent(
 	return serviceName
 }
 
-func (pm *ProcessManager) SymbolizationComplete(traceCaptureKTime times.KTime) {
```

Comment:
Moved to `processinfo.go` for consistency (all `pidToProcessInfo` accessors in one place) and renamed to `ProcessedUntil`.